docs/customer-portal/cp-cdr.md

```diff
@@ -24,3 +24,6 @@ Coordinated Universal Time (UTC) rates and stores the CDRs.
 Daily totals are also calculated in UTC.
 
 You can change the display time zone for each CDR record and saves the time zone selector, but downloads will always be in UTC.
+
+!!! Info
+    When a call enters the CDR table, you have the option to **Play Audio**. The **Call recording** is available, and you can **Download** it as well.

```

Comment:
If a recording of the call is present, you have the option to play the recording and download it as well.

Reply:
re-written the statement